### PR TITLE
f-error-message@v0.6.0 - Add aria-live attribute & tests.

### PR DESCRIPTION
**Pull request by kevinrodrigues**
## UI Review Checks

- [x] Unit tests have been [created|updated]
- [x] This code has been checked with regard to [our accessibility standards](http://fozzie.just-eat.com/documentation/general/accessibility/checklist)

## Browsers Tested

- [x] Chrome (latest)
- [ ] Internet Explorer 11
- [ ] Mobile (Please list device/browser – Ideally one iPhone model and one Android)

### List any other browsers that this PR has been tested in:

- [View the Browser Support Checklist](http://fozzie.just-eat.com/documentation/general/browser-support)

### CODE REVIEW

**Review by JoshuaNg2332 — DISMISSED** [review]

**Review by minrwhite — DISMISSED** [review]

File: packages/components/atoms/f-error-message/src/components/_tests/ErrorMessage.test.js [hunk 1]
```diff
@@ -40,4 +40,15 @@ describe('ErrorMessage', () => {
...
+
+    it('should contain the `aria-live="assertive"` attribute so screen readers can read back errors correctly', () => {
+        // Arrange
+        const wrapper = shallowMount(ErrorMessage, propsData);
+
+        // Act
+        const content = wrapper.find('[data-test-id="content"]');
+
+        // Assert
+        expect(content.attributes('aria-live')).toMatchSnapshot();
```
Comment (dandel10n) [inline packages/components/atoms/f-error-message/src/components/_tests/ErrorMessage.test.js]: Would be the check for correct attribute more effective with:
```suggestion
        expect(content.attributes('aria-live')).toBe('assertive');
```

File: packages/components/atoms/f-error-message/src/components/ErrorMessage.vue [hunk 2]
```diff
@@ -6,6 +6,7 @@
         <danger-icon :class="$style['c-errorMessage-icon']" />
         <span
             data-test-id="content"
+            aria-live="assertive"
```
Comment (dandel10n) [inline packages/components/atoms/f-error-message/src/components/ErrorMessage.vue]: Have you tried `role="alert"` instead? I am just thinking that it is more semantic to use role for the element first and only if there is no appropriate role than to use aria-live

File: packages/components/atoms/f-error-message/src/components/_tests/ErrorMessage.test.js (hunk 1 above)
Comment (kevinrodrigues) [reply packages/components/atoms/f-error-message/src/components/_tests/ErrorMessage.test.js]: Done.

File: packages/components/atoms/f-error-message/src/components/ErrorMessage.vue (hunk 2 above)
Comment (kevinrodrigues) [reply packages/components/atoms/f-error-message/src/components/ErrorMessage.vue]: Agree, fixed.

**Review by dandel10n — APPROVED** [review]

**Review by JoshuaNg2332 — APPROVED** [review]

**Review by minrwhite — APPROVED** [review]

**Pull request merged**